docusaurus.config.ts

```diff
@@ -126,8 +126,8 @@ const config: Config = {
       copyright: `Copyright © ${new Date().getFullYear()} Meroxa, Inc.`,
     },
     announcementBar: {
-      id: 'announcement-bar-1', // increment on change
-      content: `Conduit 0.8 is here! <a class='cta' href='https://github.com/ConduitIO/conduit/releases/latest' target='_blank' rel='noreferrer noopener'>See what's new</a>.`,
+      id: 'announcement-bar-2', // increment on change
+      content: `Conduit 0.9 is here! <a class='cta' href='https://github.com/ConduitIO/conduit/releases/latest' target='_blank' rel='noreferrer noopener'>See what's new</a>.`,
```

Comment:
can we also update `tar zxvf conduit_0.8.0_Darwin_x86_64.tar.gz` to `0.9` please
it's under `docs/introduction/getting-started.mdx`

Reply:
You're right about that, we need that too. I commented that on Simon's PR for the getting started guide (since I believe he'll merge it around the same time).